pkg/kubelet/prober/prober_manager.go

```diff
@@ -299,6 +301,11 @@ func (m *manager) UpdatePodStatus(podUID types.UID, podStatus *v1.PodStatus) {
 			ready = true
 		}
 		podStatus.InitContainerStatuses[i].Ready = ready
```

Comment:
can you please fix the comment above this: 

> // init containers are ready if they have exited with success or if a readiness probe has
> // succeeded.

It looks like the assumption here is that init containers has readiness probes

Reply:
I won't blame Clayton on that one :)